src/pages/MapPage.js

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { MapCenterContext } from '../context/MapCenterContext';
+import { Map } from '../components/Map';
+import { LocationsContext } from '../context/LocationsContext';
+import Spinner from '../components/Spinner';
+import ErrorMessage from '../components/ErrorMessage';
+
+const MapPage = () => {
+  const valueCenterMap = useContext(MapCenterContext);
+  const valueLocations = useContext(LocationsContext);
+  const { defaultCenterMap, newCenterMap } = valueCenterMap;
+  const { locations, status, error } = valueLocations;
+
+  return (
+    <>
+      <h1>What's Near Me?</h1>
+      <div className="view__content">
+        {status === 'loading' && <Spinner />}
```

Comment:
Why have you changed the code here? 😄

Reply:
Katia: because we needed to have a `Map` isolate component to manage more easily your events and parameter, and a page where we can use this component `MapPage`.